src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,190 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    private static Task[] commandStorage = new Task[100];
+
+    private static final int lengthOfTypeDeadline = 8;
+    private static final int lengthOfTypeToDo = 4;
+    private static final int lengthOfTypeEvent = 5;
+
+    public static void main(String[] args) {
+        printGreet();
+        receiveCommand();
+    }
+
+    private static void printGreet() {
+        String logo = " ____        _        \n"
```

Comment:
Although not necessary, the `logo` here can be factored out as a `CONSTANT` variable.